src/Tool-DependencyAnalyser/DAMessageSendAnalyzer.class.st

```diff
@@ -130,7 +130,11 @@ DAMessageSendAnalyzer >> possibleDeadCode [
 
 { #category : #accessing }
 DAMessageSendAnalyzer >> rPackage [
-	^ packageAnalysis rPackageSet packages detect: [ :each | each packageName = self packageName ]
+	"Answer a <RPackage> matching the receiver's package name. If we are browsing an unloaded package, then answer a virtual RPackage so the message send analyzer could preserve its behavior with loaded packages in #manuallyResolvedDependencies"
```

Comment:
good catch!

This is the kind of thing that a type inferencer should be able to catch super easily...

Reply:
This fix is not in the PR anymre?

Reply:
I don't know what happened, but I retrieved the changes and re-commited again